spyder/widgets/ipythonconsole/shell.py

```diff
@@ -70,6 +71,9 @@ def __init__(self, ipyclient, additional_options, interpreter_versions,
         # To save kernel replies in silent execution
         self._kernel_reply = None
 
+        color_scheme = kw['config']['JupyterWidget']['syntax_style']
+        self.set_bracket_matcher_color_scheme(color_scheme)
```

Comment:
Maybe this two lines could be replaced by something like
```python
self.set_bracket_matcher_color_scheme(self.syntax_style)
```
What do you guys think @jnsebgosselin @ccordoba12 ? 

Also, please add a comment above, something like `# To set the color of the matched parentheses. `